Oleksii_Fedorchuk/HW9_Oleksii_Fedorchuk /task1.py

```diff
@@ -0,0 +1,22 @@
+def decor(func: callable):
```

Comment:
just for the future. The naming of decorator should describe what it does. from the name decor it is not exactly clear what it is for.